lib/helpers/query/cast$expr.js

```diff
@@ -118,6 +117,16 @@ function _castExpression(val, schema, strictQuery) {
   if (val.$size) {
     val.$size = castNumberOperator(val.$size, schema, strictQuery);
   }
+  if (val.$round) {
+    const $round = val.$round;
+    if (!Array.isArray($round) || $round.length !== 2) {
```

Comment:
isnt `<place>` (the second element in the array) optional as per the docs, or am i misunderstanding this condition (and later setting)?

> <place> | integer | Optional Can be any valid [expression](https://www.mongodb.com/docs/manual/meta/aggregation-quick-reference/#std-label-aggregation-expressions) that resolves to an integer between -20 and 100, exclusive. e.g. -20 < place < 100. Defaults to 0 if unspecified.

[source](https://www.mongodb.com/docs/manual/reference/operator/aggregation/round/)